### PR TITLE
Docs edit

### PR DESCRIPTION
**Pull request by Frostedfox**
updated docs for version 3

### CODE REVIEW

File: docs/docs/contributors.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (leisim) [inline docs/docs/contributors.md:1]: The contributors get updated automatically by the `@all-contributors` bot. This document will get out of sync :/

**Review by leisim — CHANGES_REQUESTED** [review]
Thank you very much!

**Comment by leisim** [comment]
@all-contributors please add @Frostedfox for docs

**Comment by Frostedfox** [comment]
pushed corrections

**Comment by leisim** [comment]
Thanks so much!

File: docs/docs/faq.md [hunk 2]
```diff
@@ -16,6 +16,43 @@ Even today, it is very common to have no data connection, if you are in a subway
 
 The answer to this one is easy: do you have structured data or do you want to query your data? Use Isar. For dynamic data or very simple use cases, you can use Hive.
 
+### Isar
+
+> How much data can be stored and handled by Isar?
+
+That highly depends on the your objects, your indexes and which queries you want to run 🙂 everything in the low millions should be fine
```
Comment (cgestes) [inline docs/docs/faq.md:23]: missing dot? same on the next sentence?

File: docs/docs/dart.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: Dart
+---
+
+# Dart
+
+If you want to use Isar database in unit tests or Dart code, call
```
Comment (cgestes) [inline docs/docs/dart.md:7]: ```call:``` ?

**Review by cgestes — COMMENTED** [review]
a little late and a bit nitpicking :)

(I am eager to switch to the last version of isar :))

**Pull request merged**